main.nf

```diff
@@ -74,6 +74,23 @@ workflow.onError {
     NfcoreSchema.validateParameters(params, json_schema, log)
 }
 
-////////////////////////////////////////////////////
-/* --                  THE END                 -- */
-////////////////////////////////////////////////////
+def checkHostname() {
```

Comment:
I think this is only for DSL1 pipelines, there is already a check for hostnames in `lib/Checks.groovy` https://github.com/nf-core/ampliseq/blob/ffb2fa5d65bf33b4937a2f4c2f8d080776e0cba5/lib/Checks.groovy#L49

Reply:
Ah haven't seen, good catch, will test if it throws an error when I remove it!